en-US/emails/2018/firefox_learning_journey.lang

```diff
@@ -0,0 +1,344 @@
+Extensions are like toppings for your baked potato — once you try them, you can't imagine going without them. Firefoxers use extensions to give their privacy a boost, play games, block ads, manage video downloads and all kinds of other extraordinary stuff.
+
+
+;Give your browser superpowers and discover the top Firefox extensions:
+Give your browser superpowers and discover the top Firefox extensions:
+
+
+# CTA
+;See the top extensions
+See the top extensions
+
+
+# Email: Pocket
+# Mockup image: https://goo.gl/57ZQoR
+# Subject
+;If you use bookmarks, you’re gonna love this
+If you use bookmarks, you’re gonna love this
+
+
+# Pre-header
+;Introducing Pocket — a better way to bookmark.
+Introducing Pocket — a better way to bookmark.
+
+
+# Headline
+;Sure, you COULD have a million tabs open or an inbox full of articles you emailed yourself. Or you could just click Pocket.
+Sure, you COULD have a million tabs open or an inbox full of articles you emailed yourself. Or you could just click Pocket.
+
+
+;Pocket saves web pages for you to read later, without ads. It's like bookmarks, but better. And it's built right into Firefox.
+Pocket saves web pages for you to read later, without ads. It's like bookmarks, but better. And it's built right into Firefox.
+
+
+;Click that little button in the toolbar any time you find a web page you like: a tasty-looking recipe, a smart op-ed or instructions on how to build a spaceship out of twine and paper clips. Use Pocket now to save that tab you've had open since the new year.
+Click that little button in the toolbar any time you find a web page you like: a tasty-looking recipe, a smart op-ed or instructions on how to build a spaceship out of twine and paper clips. Use Pocket now to save that tab you've had open since the new year.
+
+
+;Once in Pocket, easily access it anywhere without distractions, like web banners and auto-play videos. Read, watch or even listen to articles on your phone when you have a long commute or spotty Wi-Fi. Save it now on your laptop and read it later on <a>iPhone</a>, <a>Android</a> and <a>Kindle Fire</a>.
+Once in Pocket, easily access it anywhere without distractions, like web banners and auto-play videos. Read, watch or even listen to articles on your phone when you have a long commute or spotty Wi-Fi. Save it now on your laptop and read it later on <a>iPhone</a>, <a>Android</a> and <a>Kindle Fire</a>.
+
+
+# CTA
+;Try Pocket now
+Try Pocket now
+
+
+# Closing copy
+;Pocket makes the best of the web available wherever and whenever you want — no distractions, no noise. We can't wait for you to try it out.
+Pocket makes the best of the web available wherever and whenever you want — no distractions, no noise. We can't wait for you to try it out.
+
+
+# Closing copy
+;To a better web,
+To a better web,
```

Comment:
I wonder if splitting this is going to make translation harder. Maybe it should just be `To a better web,<br>The Firefox Team`, with a note about the line break, and replace that with a new line when creating the email?